tasks/src/sentence-similarity/data.ts

```diff
@@ -74,7 +74,20 @@ const taskData: TaskDataCustom = {
 			id:          "clips/mfaq",
 		},
 	],
-	spaces:       [],
+	spaces:       [
```

Comment:
https://huggingface.co/spaces/sentence-transformers/Sentence_Transformers_for_semantic_search would be an amazing guide here